security/monitoring.adoc

```diff
@@ -0,0 +1,60 @@
+[[security-monitoring]]
+= Monitoring cluster events and logs
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[security-monitoring-intro]]
+== Introduction
+
+In addition to security measures mentioned in other sections of this guide, the ability to monitor and audit an {product-title} cluster is an important part of safeguarding the cluster itself and its users against inappropriate usage.
+
+There are two main sources of cluster-level information that administrators can use: events and logs.
+
+[[security-monitoring-events]]
+== Cluster events
+
+Cluster administrators are encouraged to familiarize themselves with the _Event_ resource type and review a xref:../dev_guide/events.adoc#events-reference[comprehensive list of events]. When monitoring or auditing a cluster, pay special attention to *node* and *system* events.
+
+Events related to resource creation, modification or deletion are good candidates to detect misuse of the cluster.
+Other events may also indicate inappropriate read-only access to protected data.
+
+
+[[security-monitoring-logs]]
+== Cluster logs
+
+Use `*journalctl*`
+
+- API Server logs
+- Kubelet logs
+- Proxy logs
+
+Link back to logging stack
+
+Others: build logs, deployment logs, ...
+
+- How to discover malicious intent from logs(?)
+
+
+== What to look for
+
+// How to tie API calls back to cluster users? Via Token?
+- Top API Server consumers
+- API requests by IP: high usage from a single IP may reveal a misbehaving user or misconfigured service
```

Comment:
IP is likely to be a load balancer or proxy and not indicate much about original user. audit log by user is probably the direction to go.